### ISSUE TITLE
[Windows] comtypes not included

### ISSUE THREAD

**Issue by mirober** [talonvoice/talon#216]
Doesn't seem to be present in site-packages and pip install fails with
```
ERROR: Error [WinError 2] The system cannot find the file specified while executing command python setup.py egg_info
ERROR: Could not install packages due to an EnvironmentError: [WinError 2] The system cannot find the file specified
```

**Comment by lunixbochs** [comment]
What's your talon version?

**Comment by mirober** [comment]
One of these days i will remember to include that.. `v0.1.4-44`

**Comment by lunixbochs** [comment]
wait are you able to use win32com instead of comtypes? we do have the former I think

**Comment by mirober** [comment]
I'll look into it, though from what I've seen projects which use Windows accessibility APIs tend to require comtypes - see e.g. [nvda](https://github.com/nvaccess/nvda/tree/master/source).

**Comment by lunixbochs** [comment]
pip should be fixed in beta 0.1.4-206
